packages/superset-ui-plugin-chart-table/src/Table.tsx

```diff
@@ -58,6 +58,17 @@ function getCellHash(cell: Cell) {
   return `${cell.key}#${cell.value}`;
 }
 
+function getText(value: string | number) {
+  if (typeof value === 'string') {
+    const span = document.createElement('span');
+    span.innerHTML = value;
```

Comment:
I would feel better if we sanitize `value` first. 
https://www.npmjs.com/package/sanitize-html

Reply:
Or there might be a way to use part of interweave to do it.
https://github.com/milesj/interweave/blob/master/packages/core/src/Parser.ts

Reply:
In the end, I chose dompurify to do this, since for the legacy tablevis, this library is used, and this library is also the smallest among several sanitization libraries.